packages/coordinator/src/components/Popup/Popup.tsx

```diff
@@ -0,0 +1,68 @@
+import { noop } from "lodash";
+import {
+  Dialog,
+  DialogTitle,
+  DialogActions,
+  DialogContent,
+  makeStyles,
+} from "@material-ui/core";
+import Button, { ButtonVariant } from "../Button";
+
+export type PopupProps = {
+  open: boolean;
+  title?: string | React.ReactNode;
+  content?: string | React.ReactNode;
```

Comment:
Do we currently need any title / content that is not string?

If not, I would keep it string for now, and change to ReactNode if we ever need it